bolt/src/DataReader.cpp

```diff
@@ -945,7 +955,10 @@ ErrorOr<Location> DataReader::parseLocation(char EndChar,
   consumeAllRemainingFS();
 
   // Read the string containing the symbol or the DSO name
-  ErrorOr<StringRef> NameRes = parseString(FieldSeparator);
+  if (!expectAndConsumeNS())
```

Comment:
If it is necessary we can remove this error and  use parseString(NameSeparator) if NameSeparator was found and FieldSeparator if it was not. But if it possible for for you @rafaelauler to replace old fdata you use without quotes with new one - I suggest just to use new variant.